scripts/lldbinit.py

```diff
@@ -44,13 +44,14 @@ def print_usage() -> None:
 Dumps the value of an associated ID, using the C++ Dump() functions.
 
 Usage:
-  dump <CONTEXT> [<EXPR>|-- <EXPR>|<TYPE><ID>|<TYPE> <ID>]
+  dump <CONTEXT> [<EXPR>|-- <EXPR>|<TYPE><ID>|<IR>.<TYPE><ID>]
 
 Args:
   CONTEXT is the dump context, such a SemIR::Context reference, a SemIR::File,
           a Parse::Context, or a Lex::TokenizeBuffer.
   EXPR is a C++ expression such as a variable name. Use `--` to prevent it from
        being treated as a TYPE and ID.
+  IR is the CheckIRId(N) in the form `irN`.
```

Comment:
Should this mention that it only supports `inst`? That seems new, and doesn't appear to be otherwise documented.

Reply:
That will become wrong in the future, so I don't feel that we should. Maybe I can say something about what the dumped ids look like instead.

Reply:
Perhaps still document that it's a subset, and where to look for the supported list? Or change the approach, and just drop the IR on the floor when it's not used?

Right now reading `TYPE` documentation says everything is allowed, so as long as it's a subset I think it's worth clarifying that the disagreement between `TYPE` documentation and behavior.